_inc/client/components/dash-item/index.jsx

```diff
@@ -106,19 +96,6 @@ export class DashItem extends Component {
 				}
 			}
 
-			if ( 'monitor' === this.props.module ) {
-				toggle = ! this.props.isDevMode && this.props.getOptionValue( this.props.module ) && (
-					<Button
-						onClick={ trackMonitorSettingsClick }
-						href={ 'https://wordpress.com/settings/security/' + this.props.siteRawUrl }
-						compact>
-						{
-							__( 'Settings' )
-						}
-					</Button>
-				);
-			}
-
```

Comment:
This was the "settings" button at the dashboard. Switching it to a toggle instead.